include/commonpp/metric/type/DescStat.hpp

```diff
@@ -138,7 +144,13 @@ class DescStat : private boost::noncopyable
         result.push(weighted_mean(acc), "mean");
         result.push(min(acc), "min");
         result.push(max(acc), "max");
-        result.push(weighted_variance(acc), "variance");
+        {
+            auto var = weighted_variance(acc);
+            if(var >= 0)
+            {
+                result.push(::sqrt(var), "stdev");
```

Comment:
this could have been done directly in the time series database but fair enough :)

Reply:
This is the standard deviation of the underlying data. The time series database only sees the aggregated metrics, so it cannot compute this.

Reply:
I meant the sqrt, the variance is enough to calculate the stdev on the db itself, but the opposite is also true, we can compute the variant from the stdev.